package_test.go

```diff
@@ -0,0 +1,156 @@
+package sqlair_test
+
+import (
+	"database/sql"
+	"testing"
+
+	_ "github.com/mattn/go-sqlite3"
+	. "gopkg.in/check.v1"
+
+	"github.com/canonical/sqlair"
+)
+
+// Hook up gocheck into the "go test" runner.
+func TestExpr(t *testing.T) { TestingT(t) }
+
+type PackageSuite struct{}
+
+var _ = Suite(&PackageSuite{})
+
+func setupDB() (*sql.DB, error) {
+	return sql.Open("sqlite3", ":memory:")
+}
+
+func createExampleDB(create string, inserts []string) (*sql.DB, error) {
+	db, err := setupDB()
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = db.Exec(create)
+	if err != nil {
+		return nil, err
+	}
+	for _, insert := range inserts {
+		_, err := db.Exec(insert)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	return db, nil
+}
+
+type JujuLeaseKey struct {
+	Namespace string `db:"type"`
+	ModelUUID string `db:"model_uuid"`
+	Lease     string `db:"name"`
+}
+
+type JujuLeaseInfo struct {
+	Holder string `db:"holder"`
+	Expiry int    `db:"expiry"`
+}
+
+func JujuStoreLeaseDB() (string, *sql.DB, error) {
+	create := `
+CREATE TABLE lease (
+	model_uuid text,
+	name text,
+	holder text,
+	expiry integer,
+	lease_type_id text
+);
+CREATE TABLE lease_type (
+	id text,
+	type text
+);
+
+`
+	drop := `
+	 drop table lease;
```

Comment:
As in the above comment, perhaps we could align the tabs according to those of `create` above here as well.

Reply:
Changed.